src/utils_orders.py

```diff
@@ -0,0 +1,311 @@
+import logging
+import os
+import re
+import pycountry
+import urllib.parse
+from datetime import datetime
+from oms_jira import MPClient
+from oms_jira.services.mp import MPMessage, ScopeEnum, MessageAuthor, ProjectItemStatusEnum
+from waldur_client import WaldurClient, ProjectRole
+
+logging.getLogger("requests").setLevel(logging.WARNING)
+logging.basicConfig(level=logging.INFO, format=f'[%(asctime)s] %(filename)s:%(lineno)d %(levelname)s - '
+                                               f'%(message)s')
+
+EOSC_URL = os.environ.get('EOSC_URL')  # polling url
+TOKEN = os.environ.get('TOKEN')
+OMS_ID = os.environ.get('OMS_ID')
+
+USERNAME = os.environ.get('USERNAME')
+PASSWORD = os.environ.get('PASSWORD')
+WALDUR_URL = os.environ.get('WALDUR_URL')
+WALDUR_API = urllib.parse.urljoin(WALDUR_URL, 'api/')
+WALDUR_API_AUTH = urllib.parse.urljoin(WALDUR_URL, 'api-auth/password/')
+
+WALDUR_TARGET_ORGANIZATION_NAME = os.environ.get('WALDUR_TARGET_ORGANIZATION_NAME', 'Test4All')
+
+WALDUR_TOKEN = os.environ.get('WALDUR_TOKEN')
+
+# def get_waldur_token(): #
```

Comment:
Do we still need this token?

Reply:
will keep 29-34 function to test with local waldur